src/utils/exec.h

```diff
@@ -60,6 +60,7 @@ gint bd_utils_version_cmp (const gchar *ver_string1, const gchar *ver_string2, G
 gboolean bd_utils_check_util_version (const gchar *util, const gchar *version, const gchar *version_arg, const gchar *version_regexp, GError **error);
 
 gboolean bd_utils_init_prog_reporting (BDUtilsProgFunc new_prog_func, GError **error);
+gboolean bd_utils_prog_reporting_initialized();
```

Comment:
Please add space between the function name and parentheses (and in the definition too). Thanks